src/main/java/com/aws/greengrass/logmanager/CloudWatchAttemptLogsProcessor.java

```diff
@@ -122,15 +124,22 @@ public CloudWatchAttempt processLogFiles(ComponentLogFileInformation componentLo
                 .replace("{componentName}", componentLogFileInformation.getName());
         attempt.setLogGroupName(logGroupName);
         String logStreamName = getLogStreamName(thingName);
-
+        LogFileGroup logFileGroup = componentLogFileInformation.getLogFileGroup();
         // Run the loop until all the log files from the component have been read or the max message
         // size has been reached.
         while (!componentLogFileInformation.getLogFileInformationList().isEmpty() && !reachedMaxSize.get()) {
-            LogFile logFile = componentLogFileInformation.getLogFileInformationList().get(0).getLogFile();
-            long startPosition = componentLogFileInformation.getLogFileInformationList().get(0).getStartPosition();
-            //TODO Optional<String> fileHash = componentLogFileInformation.getLogFileInformationList().get(0)
-            // .getFileHash();
+            LogFileInformation tempLogFileInfomation = componentLogFileInformation.getLogFileInformationList().get(0);
```

Comment:
`temp` is not necessary. Just call it the `logFileInformation`
Also typo: `Infomation` --> `Information`